salt/soc/defaults.yaml

```diff
@@ -1717,23 +1717,23 @@ soc:
               showSubtitle: true
             - name: HTTP
               description: HTTP grouped by destination port
-              query: 'tags:(http OR http2) | groupby destination.port'
+              query: 'tags:http OR tags:http2 | groupby destination.port'
               showSubtitle: true
             - name: HTTP
               description: HTTP grouped by status code and message
-              query: 'tags:(http OR http2) | groupby http.status_code http.status_message'
+              query: 'tags:http OR tags:http2 | groupby http.status_code http.status_message'
               showSubtitle: true
             - name: HTTP
               description: HTTP grouped by method and user agent
-              query: 'tags:(http OR http2) | groupby http.method http.useragent'
+              query: 'tags:http OR tags:http2 | groupby http.method http.useragent'
               showSubtitle: true
             - name: HTTP
               description: HTTP grouped by virtual host
-              query: 'tags:(http OR http2) | groupby http.virtual_host'
+              query: 'tags:http OR tags:http2 | groupby http.virtual_host'
               showSubtitle: true
             - name: HTTP
               description: HTTP with exe downloads
-              query: 'tags:(http OR http2) AND file.resp_mime_types:*exec* | groupby http.virtual_host'
+              query: 'tags:http OR tags:http2 AND file.resp_mime_types:*exec* | groupby http.virtual_host'
```

Comment:
Should this be the following?

(tags:http OR tags:http2) AND file.resp_mime_types:*exec* | groupby http.virtual_host